File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/LoginSuccessResponse.js
goog.provide('API.Client.LoginSuccessResponse');

/**
 * The response from a successful login.
 * @record
 */
API.Client.LoginSuccessResponse = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.LoginSuccessResponse.prototype.sessionId;

/**
 * @type {!number}
 * @export
 */
API.Client.LoginSuccessResponse.prototype.accountId;

/**
 * @type {!string}
 * @export
 */
API.Client.LoginSuccessResponse.prototype.accountLid;

/**
 * @type {!string}
 * @export
 */
API.Client.LoginSuccessResponse.prototype.ima;

/**
 * @type {!string}
 * @export
 */
API.Client.LoginSuccessResponse.prototype.gravatar;

File: OPENAPI-REST-API/openapi-client/clojure/src/inter_server_management_api/specs/login_success_response.clj
(ns inter-server-management-api.specs.login-success-response
  (:require [clojure.spec.alpha :as s]
            [spec-tools.data-spec :as ds]
            )
  (:import (java.io File)))


(def login-success-response-data
  {
   (ds/opt :sessionId) string?
   (ds/opt :account_id) int?
   (ds/opt :account_lid) string?
   (ds/opt :ima) string?
   (ds/opt :gravatar) string?
   })

(def login-success-response-spec
  (ds/spec
    {:name ::login-success-response
     :spec login-success-response-data}))
